server/middleware/loan-middleware.js

```diff
@@ -1,92 +1,72 @@
+import sgMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
 import db from '../database/config-db';
 import queries from '../database/queries-db';
 
+dotenv.config();
+
 const middleware = {
-  async checkEligibility(req, res, next) {
-    const { rows } = await db(queries.checkRepaid(req.user.email));
+  async checkVerified(req, res, next) {
+    const { rows } = await db(queries.checkVerification(req.user.email, 'unverified'));
     if (rows.length) {
       return res.status(409).json({
         status: 409,
```

Comment:
I'm not sure this is a 409. The user is trying to make an action he/she is unauthorized to make at the moment.

Reply:
Yes, thank you i'll make changes to that

Reply:
Thank you for noting that, I've implemented it...